website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -78,22 +88,43 @@ Options:
       --network-file <path>     Networks config file path (default: "./networks.json")
 ```
 
-The `add` command will fetch the ABI from Etherscan (unless an ABI path is specified with the `--abi` option), and will create a new `dataSource` in the same way that `graph init` command creates a `dataSource` `--from-contract`, updating the schema and mappings accordingly.
+#### Specifics
 
-The `--merge-entities` option identifies how the developer would like to handle `entity` and `event` name conflicts:
+The `add` command will fetch the ABI from Etherscan unless an ABI path is specified with the `--abi` option. Subsequently, it will create a new `dataSource` like `graph init` command creates a dataSource `--from-contract` and will update the schema and mappings accordingly. This allows you to index implementation contracts from their proxy contracts.
```

Comment:
There's something wonky in this sentence and I'm not sure how to fix it:

> Subsequently, it will create a new `dataSource` like `graph init` command creates a dataSource `--from-contract` and will update the schema and mappings accordingly.